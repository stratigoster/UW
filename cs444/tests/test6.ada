PACKAGE foo IS
  i : INTEGER := 3+4*5;
  s : STRING := "Bye World""";
  b2 : INTEGER := 5 + j + i + i * 2 / 3;
--                    ^
-- undeclared variable j
END foo;
