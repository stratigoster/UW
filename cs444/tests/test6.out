***ERROR***:4:26:	Undeclared variable ` j '

------
Compilation error
------
